### ISSUE TITLE
Foxx queue: job isn't running when server.authentication = true

### ISSUE THREAD

**Issue by mpv1989** [arangodb/arangodb#3120]
## my environment running ArangoDB
I'm using the latest ArangoDB of the respective release series:
- [x] 3.2.1
- [x] self-compiled devel branch

Mode:
- [x] Single-Server

Storage-Engine:
- [x] mmfiles
- [x] rocksdb

On this operating system:
- [ ] Linux 
  - [x] Ubuntu .deb
- [x] Windows, version: 10

### Foxx
Jobs in Foxx queues aren't executed when authentication is on.

### How to reproduce:
Starting arangod with `server.authentication true`, creating a queue within a Foxx service and pushing a job to it.

I also created a branch with a failing test: [https://github.com/arangodb/arangodb/tree/bug-fix/foxx-queue-auth](https://github.com/arangodb/arangodb/tree/bug-fix/foxx-queue-auth)

**Comment by joewestcott** [comment]
Can also replicate on OSX & 3.2.1, very frustrating.
 I've not changed `server.authentication true` (to my knowledge), so unless it defaults to `true` it might not be related...?

**Comment by mpv1989** [comment]
@joewestcott `server.authentication true` is default.

we already found the bug. pull request [here](https://github.com/arangodb/arangodb/pull/3122)

**Comment by mpv1989** [comment]
[PR 3.2](https://github.com/arangodb/arangodb/pull/3134)
[PR devel](https://github.com/arangodb/arangodb/pull/3122)

**Comment by dothebart** [comment]
ArangoDB 3.2.3 is now available for download. Please note that we switched our webserver and changed our download URLS for better performance.
Closing.

**Comment by kwando** [comment]
This is still a problem on the latest version available on brew..

```
2018-02-15T22:29:09Z [30135] INFO ArangoDB 3.3.3 [darwin] 64bit, using jemalloc, VPack 0.1.30, RocksDB 5.6.0, ICU 58.1, V8 5.7.492.77, OpenSSL 1.0.2n  7 Dec 2017
2018-02-15T22:29:09Z [30135] INFO using storage engine mmfiles
2018-02-15T22:29:09Z [30135] INFO {cluster} Starting up with role SINGLE
2018-02-15T22:29:09Z [30135] INFO {syscall} file-descriptors (nofiles) hard limit is unlimited, soft limit is 8192
2018-02-15T22:29:09Z [30135] INFO {authentication} Authentication is turned off, authentication for unix sockets is turned on
2018-02-15T22:29:09Z [30135] INFO running WAL recovery (2 logfiles)
2018-02-15T22:29:09Z [30135] INFO replaying WAL logfile '/usr/local/var/lib/arangodb3/journals/logfile-246353.db' (1 of 2)
2018-02-15T22:29:09Z [30135] INFO replaying WAL logfile '/usr/local/var/lib/arangodb3/journals/logfile-246377.db' (2 of 2)
2018-02-15T22:29:09Z [30135] INFO WAL recovery finished successfully
2018-02-15T22:29:09Z [30135] INFO using endpoint 'http+tcp://127.0.0.1:8529' for non-encrypted requests
2018-02-15T22:29:11Z [30135] INFO ArangoDB (version 3.3.3 [darwin]) is ready for business. Have fun!
```

**Comment by OmarAyo** [comment]
Hi @kwando 

ArangoDB 3.3.9 is now available for download 

Please let us know how things are with this version 

Best,

**Comment by OmarAyo** [comment]
Hi @kwando 

FYI version 3.2.15 and 3.3.10 are now available, as many bugs were fixed from 3.2.1, so we strongly suggest you upgrade to the latest version and see if the problem still occurs

Best,

**Comment by OmarAyo** [comment]
I am going to close this ticket. Tickets can be reopened at any time by adding a new comment

Best
